checkov/terraform/checks/resource/ncp/AccessControlGroupOutboundRule.py

```diff
@@ -0,0 +1,25 @@
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+from checkov.common.models.enums import CheckResult, CheckCategories
+
+
+class AccessControlGroupOutboundRule(BaseResourceCheck):
+    def __init__(self):
+        name = "An outbound security group rule allows traffic to /0"
+        id = "CKV_NCP_3"
+        supported_resources = ['ncloud_access_control_group_rule']
+
+        categories = [CheckCategories.NETWORKING]
+        guideline = "You should restrict access to IP addresses or ranges that are explicitly required where possible."
+        super().__init__(name=name, id=id, categories=categories,
+                         supported_resources=supported_resources, guideline=guideline)
+
+    def scan_resource_conf(self, conf):
+        if 'outbound' in conf.keys():
+            for inbound in conf['outbound']:
+                ip = inbound.get('ip_block', '0.0.0.0/0')[0]
+                if ip == '0.0.0.0/0' or ip == '::/0':
```

Comment:
```suggestion
            for outbound in conf['outbound']:
                ip = outbound.get('ip_block')
                if ip == ['0.0.0.0/0'] or ip == ['::/0']:
```
you probably meant `outbound` here 😄 
this way is a bit safer, if `ip_block` is a value, which can't identify.